lib/ex_unit/lib/ex_unit/diff.ex

```diff
@@ -269,11 +314,9 @@ defmodule ExUnit.Diff do
   defp script_keyword_inner({key, val1}, {key, val2}, true),
     do: [{:eq, format_key(key, true)}, script_inner(val1, val2)]
 
-  defp script_keyword_inner(_pair1, _pair2, true),
-    do: nil
+  defp script_keyword_inner(_pair1, _pair2, true), do: nil
 
-  defp script_keyword_inner(elem1, elem2, false),
-    do: script(elem1, elem2)
```

Comment:
You are not running the latest version of the formatter, that's why this change happened. Make sure you have 1.6.5 in your system or run `bin/elixir bin/mix format`.

Reply:
oh, yeah, sorry! My vim integration isn't set for `bin/elixir bin/mix` 😢